content/articles/spring-webflux/index.md

```diff
@@ -126,7 +126,7 @@ public class ApplicationConfig {
     }
 }
 ```
- - The code snippet above sets the path to the `application.properties` file, it contains the project configurations.
+ - The code snippet above sets the path to the `application.properties` file. It contains the project configurations.
 
 3. Create a new file named `MongoConfig.java` in the `config` package we created earlier and the code snippet below.
```

Comment:
Change:

> and the code snippet below.

To:

> with the code snippet below.

To show that the code belongs in that file you were talking about. The word *and* there just feels a like awkward and gives it a different meaning even though I know what you're trying to say.